build.sbt

```diff
@@ -5,19 +5,21 @@ def finagle(mod: String) =
   "com.twitter" %% s"finagle-$mod" % "7.1.0"
 
 def telemetry(mod: String) =
-  "io.buoyant" %% s"telemetry-$mod" % "1.3.3"
+  "io.buoyant" %% s"telemetry-$mod" % linkerdVersion
 
 def zipkin(mod: String) =
   "io.zipkin.finagle" %% s"zipkin-finagle-$mod" % "1.1.0"
 
 def scalatest() =
   "org.scalatest" %% "scalatest" % "3.0.1"
 
+val linkerdVersion = "1.3.3"
```

Comment:
does this need to go above `telemetry()` ?

Reply:
yeah, fixed